app/tests/spec/models/user.js

```diff
@@ -190,11 +191,12 @@ define(function (require, exports, module) {
 
     it('clearSignedInAccount', function () {
       sinon.spy(notifier, 'triggerRemote');
-      return user.setSignedInAccount({ email: 'email', uid: 'uid' })
+      const uid = createUid();
+      return user.setSignedInAccount({ email: 'email', uid })
```

Comment:
I tried validating `email` on the Account model and indeed broke a bunch of tests. I've opened https://github.com/mozilla/fxa-content-server/issues/4993 to validate Account data as it comes in. In the meantime, I've updated all the `email` fields in this module to be valid email addresses. Thanks @vbudhram.